facia/public/facebook_ia_remote.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex">
+    <script>
+    </script>
+</head>
+<body style="margin:0">
+<div id="c" style="position:absolute;top:0;left:0;bottom:0;right:0;">
+
+    <script>
+        (function () {
+
+            function parseParameters() {
+                var script = document.createElement("script");
+                script.id = "ad-slot-id";
+
+                var args = window.location.hash.substr(1).split('&');
+
+                for (var i = 0; i < args.length; ++i) {
```

Comment:
I think it is best to declare the variables outside the for loop. Like this:

```
var i = 0;
var argLength = args.length;

for (i = 0; i < argLength; i++) ...
```

Currently, args.length is being re-evaluated each time this loop is run.